static/js/click_handlers.js

```diff
@@ -988,8 +988,13 @@ export function initialize() {
         }
 
         if (compose_state.composing()) {
+            const $closest_links = $(e.target).closest("a");
+            const nearby_link_exists = $closest_links.length > 0;
+            // Links that are actually dropdown widget don't count.
+            const inside_dropdown_widget =
+                nearby_link_exists && $closest_links[0].role === "menuitem";
```

Comment:
I think this might have the wrong specificity. The code below is intended to refocus the compose box when doing clicks in the message feed... basically to override the browser's default behavior that clicking a link in the message feed will move focus to that element, which prevents you from typing more content into the compose box afterwards.

I guess we need to be careful about sidebar clicks too. 

It's likely a sensitive change, but probably we want something like a positive check for "Element is inside the main message feed and sidebar containers", not an exclude for menu items?

Reply:
I'm a little nervous about doing a positive check -- do you feel confident that we could manually test that well enough to not break things?

Reply:
Yeah I think we'll need to run this on chat.zulip.org for extended QA ... but probably any version of what we do with this code is going to involve that, and we might as well do that process with the code that we think is logically correct/maintainable.